CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.6.1 (Jul 20, 2023)
+
+IMPROVEMENTS
+* Bump Consul OSS image version to 1.16.0 and Consul enterprise version to 1.16.0-ent
+* Bump envoy image to 1.26.2 
```

Comment:
ECS 0.6.x supports 1.13, 1.14, and 1.15 so I don't think we technically should bump this to 1.16 yet due to the N-2 Consul version maintenance policy.

So set Consul to 1.15.4.

For Envoy, we can pick the [latest default version that is supported by Consul version 1.13](https://developer.hashicorp.com/consul/docs/v1.15.x/connect/proxies/envoy#supported-versions) which is Envoy 1.23.*

Reply:
Is there a compatibility matrix that we follow for ECS <-> Consul versions?

Reply:
Should we also sort the acceptance tests before pushing this change? I'd either way start making changes to them as part of the new architecture

Reply:
It would be good to fix the acceptance tests, yes. Let me know if you need help with anything there (like AWS account clean up)

Reply:
https://developer.hashicorp.com/consul/docs/ecs/compatibility#supported-consul-versions

We started out doing 1:1 Consul version to ECS version, but in 0.5.x we added acceptance tests to validate back to N-2 Consul versions for an ECS version, which really is necessary when considering upgrades, I think.

The ECS compat matrix page needs an update to show:

* 0.6.x -> 1.15, 1.14, 1.13
* 0.5.x -> 1.14, 1.13, 1.12

Reply:
Raised https://github.com/hashicorp/terraform-aws-consul-ecs/pull/186 to fix the existing tests. This PR can be validated once the other PR lands in main.